source/quickstart.txt

```diff
@@ -253,3 +253,5 @@ Synchronization Notes
 - To estimate the size of ``oplog`` needed for initial synchronization,
   see: :ref:`c2c-oplog-sizing`.
 
+.. include:: /includes/fact-reverse-sync-action-types.rst
+
```

Comment:
I'd personally place this note alongside the 2nd bullet which also talks about reversing, rather than splitting them up. We could inline the note to right under the 2nd bullet, or make that bullet the last one in the list and keep the note here -- I don't think it matters too much.